Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -1187,7 +1185,7 @@ function BuildTimeline {
                                 if ($CalLog.ResponsibleUser -eq "Calendar Assistant") {
                                     [array] $Output = "The Exchange System $($CalLog.TriggerAction)d the meeting via the Calendar Assistant."
                                 } else {
-                                    [array] $Output = "$($CalLog.TriggerAction) to the Meeting by [$($CalLog.ResponsibleUser)] with $($CalLog.Client)."
+                                    [array] $Output = "[$($CalLog.ResponsibleUser)] $($CalLog.TriggerAction)d the Meeting with $($CalLog.Client)."
```

Comment:
Do you want the `d` after `$CalLog.TriggerAction` ?

Reply:
Yes, the Trigger Action is Update, Create, etc. so the "d" puts it in the past tense.